docs/setup_guide.md

```diff
@@ -0,0 +1,120 @@
+# Setup Guide
+
+This guide will provide the steps how to setup a DAO module with a Safe on Rinkeby. The setup will use Realitio and can be used with Snapshot spaces.
+
+More information how this works can be found on the [Gnosis blog](https://blog.gnosis.pm/ea67eb95c34f).
```

Comment:
~More information how this works can be found on the~
For more information on SafeSnap please refer to the [Gnosis blog](link).